examples/grpc-bidirectional-streaming-copy/grpc_bidirectional_streaming.md

```diff
@@ -0,0 +1,54 @@
+# Bidirectional streaming RPC copy
+
+The gRPC server connector exposes the gRPC service over HTTP2. In a gRPC bidirectional streaming scenario, the gRPC service and the client operate when each other sends a sequence of messages using a read-write stream. In such scenarios, the two streams operate independently. Therefore, clients and servers can read and write in any order.
+
+>**Info:** For more information on the underlying module, see the [`grpc` module](https://lib.ballerina.io/ballerina/grpc/latest/).
+
+## Generate the service definition
+
+1. Create a new directory (e.g., `grpc_bidirectional_streaming`).
+
+2. Create a new Protocol Buffers definition file `grpc_bidirectional_streaming.proto` inside the `grpc_bidirectional_streaming` directory, and add the service definition below to it.
+
+    ::: code grpc_bidirectional_streaming.proto :::
+
+3. Navigate to the `grpc_bidirectional_streaming` directory, and run the command below for stub generation.
+
+    ::: code grpc_bidirectional_streaming.out :::
+
+Once you run the command, the `grpc_bidirectional_streaming_pb.bal` file gets generated inside the `stubs` directory. 
```

Comment:
Shouldn't we indent this to align with number 3?

Reply:
This is what we get after running all the steps and is not part of step 3.